code/nodes/saveGraph/utils.q

```diff
@@ -83,26 +86,27 @@ saveGraph.i.displayConfMatrix:{[confMatrix;classes;modelName;savePath]
 
 // @kind function
 // @category saveGraphUtility
-// @fileoverview Add text to confusion matrix
+// @desc Add text to confusion matrix
 // @param confMatrix {int[]} Confusion matrix
-// @param thresh     {int} Threshold value
-// @param i          {int} Row in the confusion matrix
-// @param j          {int} column in the confusion matrix
-// @return {null} Adds text to plot
+// @param thresh {int} Threshold value
+// @param i {int} Row in the confusion matrix
+// @param j {int} column in the confusion matrix
+// @return {::} Adds text to plot
 saveGraph.i.addText:{[confMatrix;thresh;i;j]
   color:$[thresh<confMatrix[i;j];`white;`black];
   valueStr:string confMatrix[i;j];
-  utils.plt[`:text][j;i;valueStr;`horizontalalignment pykw`center;`color pykw color]
+  utils.plt[`:text][j;i;valueStr;`horizontalalignment pykw`center;
+  `color pykw color];
```

Comment:
Double spacing needed here for continuation of line

Reply:
indentation added